src/style.css

```diff
@@ -1 +1,56 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.container {
+  height: 96px;
+
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
```

Comment:
The `flex-direction: column;` on the `.container` class might not be necessary if you intend to display the stars horizontally. Consider using `flex-direction: row;` or removing this line if the default row direction is desired.